docs/content.zh/docs/connectors/table/formats/avro-confluent.md

```diff
@@ -280,6 +280,13 @@ Format 参数
             <td>String</td>
             <td>The URL of the Confluent Schema Registry to fetch/register schemas.</td>
         </tr>
+        <tr>
+            <td><h5>auto.register.schemas</h5></td>
+            <td>optional</td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Whether to automatically register schemas with the Confluent Schema Registry if they don't exist. When set to <code>false</code>, schemas must be manually registered in the Schema Registry before being used. When set to <code>true</code>, schemas will be automatically registered during serialization if they don't already exist. The default value is <code>true</code>.</td>
```

Comment:
Did we document anywhere in which scenario we auto register schemas for example when reading or writing from/to a table?

Reply:
@fapaul I guess saying serialization implies that we creating an outbound flow with the schema id- e.g. writing to a sink. We could document this more explicitly.